app/components/post_list/post_list.tsx

```diff
@@ -130,7 +130,7 @@ const PostList = ({
             return false;
         }
         return posts[0]?.id !== lastPostId;
-    }, [posts[0]?.id, lastPostId]);
+    }, [lastPostId, posts]);
```

Comment:
Probably this memo is not needed at all (it is memoizing a boolean value and doing a pretty fast operation). But 0/5 on whether it is desirable to change it in this PR.